### ISSUE TITLE
object.Exception@src/config.d(47): Missing config value: no-remote-delete

### ISSUE THREAD

**Issue by EvoBiomech** [abraunegg/onedrive#102]
**Describe the bug**
Client fails due to missing config value - cannot be run as service.
 
**Application and Operating System Details:**
Linux david-office 4.15.0-30-generic #32-Ubuntu SMP Thu Jul 26 17:42:43 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

onedrive v2.0.2-20-gb220bf8

**Complete Verbose Log Output**
``` onedrive --verbose --synchronize
Loading config ...
Using Config Dir: /home/david/.config/onedrive
No config file found, using defaults
Initializing the OneDrive API ...
Opening the item database ...
All operations will be performed in: /home/david/OneDrive
Initializing the Synchronization Engine ...
Account Type: business
Default Drive ID: b!QoQNy4IZqU6VRI_5NQ0iSNgYwMlSTLNKhxSOuq_xgnoH6olOO-LhRrNhMIWfmwuD
Default Root ID: 01QEM7XZN6Y2GOVW7725BZO354PWSELRRZ
Remaining Free Space: 5490928996162
Fetching details for OneDrive Root
OneDrive Root exists in the database
Syncing changes from OneDrive ...
Applying changes of Path ID: 01QEM7XZN6Y2GOVW7725BZO354PWSELRRZ
Uploading differences of .
Processing root
The directory has not changed
Processing Documents
The directory has not changed
Processing Bibliography.bib.bak
The file has not changed
Processing Bibliography.bib.sav
The file has been deleted locally
Missing config value: no-remote-delete
Syncing changes from OneDrive ...
Applying changes of Path ID: 01QEM7XZN6Y2GOVW7725BZO354PWSELRRZ
Uploading differences of .
Processing root
The directory has not changed
Processing Documents
The directory has not changed
Processing Bibliography.bib.bak
The file has not changed
Processing Bibliography.bib.sav
The file has been deleted locally
Missing config value: no-remote-delete
Syncing changes from OneDrive ...
Applying changes of Path ID: 01QEM7XZN6Y2GOVW7725BZO354PWSELRRZ
Uploading differences of .
Processing root
The directory has not changed
Processing Documents
The directory has not changed
Processing Bibliography.bib.bak
The file has not changed
Processing Bibliography.bib.sav
The file has been deleted locally
object.Exception@src/config.d(47): Missing config value: no-remote-delete
----------------
src/config.d:47 immutable(char)[] config.Config.getValue(immutable(char)[]) [0x1785886c]
src/sync.d:913 void sync.SyncEngine.uploadFileDifferences(itemdb.Item, immutable(char)[]) [0x178ceb64]
src/sync.d:760 void sync.SyncEngine.uploadDifferences(itemdb.Item) [0x178cde8c]
src/sync.d:780 void sync.SyncEngine.uploadDirDifferences(itemdb.Item, immutable(char)[]) [0x178ce049]
src/sync.d:757 void sync.SyncEngine.uploadDifferences(itemdb.Item) [0x178cde59]
src/sync.d:720 void sync.SyncEngine.scanForDifferences(immutable(char)[]) [0x178cdc89]
src/main.d:398 void main.performSync(sync.SyncEngine, immutable(char)[], bool, bool, bool) [0x178b138f]
src/main.d:264 _Dmain [0x178b0cb9]
'''

**Comment by abraunegg** [comment]
Can you provide some additional information here:

1. Are you starting the application manually, if so, what is the full command line your using?
2. Are you starting the application via the systemd files, if so, have you modified the systemd unit files at all?

**Comment by MK-42** [comment]
Can confirm this problem. Happens when executing `onedrive --synchronize --verbose`
fetching 'no-remote-delete' from cfg fails because its not in the default configuration, I think.

**Comment by EvoBiomech** [comment]
I can confirm that pr103 solves the problem. Thanks for the very quick response.

**Comment by abraunegg** [comment]
Once reviewer approved, will sync this into master ASAP

**Comment by abraunegg** [comment]
merged PR #103 into master
